File: HomeMart/src/app/model/AddressModel.ts
export class City{
    public City_Id: string;
    public City_Name:string;
}

export class Districts{
    public  City_Id : string;
    public  Districts_Id : string;
    public  Districts_Name :string;
}

export class AddressModel{
    public  City_Name : string;
    public  Districts_Name : string;
    public  Address :string;
}

File: BT.API.HOME/BT.API.HOME/Models/AddressModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BT.API.HOME.Models
{
    public class AddressModel
    {
        public class City
        {
            public string City_Id { get; set; }
            public string City_Name { get; set; }
        }

        public class Districts
        {
            public string City_Id { get; set; }
            public string Districts_Id { get; set; }
            public string Districts_Name { get; set; }
        }
        public class ResultAddress
        {
            public string City_Name { get; set; }
            public string Districts_Name { get; set; }
            public string Address { get; set; }
        }
    }
}